css-four-column-layout/styles.css

```diff
@@ -0,0 +1,35 @@
+.container {
+  justify-content: center;
+  max-width: 800;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: Barlow, sans-serif;
+}
+
+.screen {
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.icon {
+  width: 20%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+img {
+  width: 100%;
+}
+
+.row-title {
+  text-align: center;
+}
+
+.row {
+  display: flex;
+}
+
+.column-fourth {
```

Comment:
make sure you are including the proper width or flex basis for your columns and not letting flexbox determine the size!